pytorch_lightning/accelerators/ddp_accelerator.py

```diff
@@ -180,7 +173,7 @@ def set_world_ranks(self, process_idx):
         self.trainer.world_size = self.trainer.num_nodes * self.trainer.num_processes
 
     def model_to_device(self, model, process_idx):
```

Comment:
do we need process_idx at all then?

Reply:
yes, we can probably remove it. the device the model is on is anyway not related to the process index, which was actually the root cause of the bug here.